### PR TITLE
Olha maraduda

### PR DESCRIPTION
**Pull request by OlyaOlyaaa**

### CODE REVIEW

**Comment by OlyaOlyaaa** [comment]
Added folder and file with HW3. Hopefully this has been done as expected.

**Comment by OlyaOlyaaa** [comment]
HW4 was added to my branch.

**Comment by OlyaOlyaaa** [comment]
HW5 was added

File: HW3_OlhaMaraduda/HW3_Olha_Maraduda.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# 48.Unless explicitly silenced.
+# 49.In the face of ambiguity, refuse the temptation to guess.
+# 50.There should be one-- and preferably only one --obvious way to do it.
+# 51.Although that way may not be obvious at first unless you're Dutch.
+# 52.Now is better than never.
+# 53.Although never is often better than *right* now.
+# 54.If the implementation is hard to explain, it's a bad idea.
+# 55.If the implementation is easy to explain, it may be a good idea.
+# 56.Namespaces are one honking great idea -- let's do more of those!"""
+
```
Comment (kolyasalubov) [inline HW3_OlhaMaraduda/HW3_Olha_Maraduda.py:62]: good job

File: HW3_OlhaMaraduda/HW3_Olha_Maraduda.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+#     Zen_of_python = file.read()
+#     Zen_of_python_lower = Zen_of_python.lower()
+#     better_count = Zen_of_python_lower.count(" better ")
+#     never_count = Zen_of_python_lower.count(" never ")
+#     is_count = Zen_of_python_lower.count ( " is ")
+
+# print(f'The word "better" appears {better_count} times in the text.')
+# print(f'The word "never" appears {never_count} times in the text.')
+# print(f'The word "is" appears {is_count} times in the text.')
+
```
Comment (kolyasalubov) [inline HW3_OlhaMaraduda/HW3_Olha_Maraduda.py:86]: good job

File: HW3_OlhaMaraduda/HW3_Olha_Maraduda.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# for digit in digits:
+#     product *= digit
+
+# reverse_number = int(str(number)[::-1])
+# sorted_digits = ''.join(sorted(str(number)))
+
+# print("Product of the digits:", product)
+# print("Reversed number:", reverse_number)
+# print("Digits sorted in ascending order:", sorted_digits)
+
```
Comment (kolyasalubov) [inline HW3_OlhaMaraduda/HW3_Olha_Maraduda.py:105]: good job

File: HW4_OlhaMaraduda/HW4_Olha_Maraduda.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# "Temperature Converter"
+
+# Write a Python program that prompts the user to enter a temperature in Celcius and then converts it to Fahrenheit
+# using the formula: F = (C * 9/5) + 32. Your program should then print out the converted te,perature in Fahrenheit.
+# However, if the user enters a temperature  below -234.15C (the lowest possible temperature in the universe, also 
+# known as absolyre zero), your program should print an error message instead of attempting to comvert the temperature.
+
+
```
Comment (kolyasalubov) [inline HW4_OlhaMaraduda/HW4_Olha_Maraduda.py:8]: excellent work

File: HW5_OlhaMaraduda/HW5_Task1_Olha_Maraduda.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# TASK_1
+# Create a list that contains elements of integer type, then use the loop to change the type of these elements
+# to a floating type. (Hint: use built-in float() function)
+
```
Comment (kolyasalubov) [inline HW5_OlhaMaraduda/HW5_Task1_Olha_Maraduda.py:4]: good job

File: HW5_OlhaMaraduda/HW5_Task2_Olha_Maraduda.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# TASK2
+# Print Fibonacci numbers up to the entered number n, using cycles.
+# (Sequence of Fibonacci numbers 0, 1, 1, 2, 3, 5, 8, 13 etc.)
+
```
Comment (kolyasalubov) [inline HW5_OlhaMaraduda/HW5_Task2_Olha_Maraduda.py:4]: excellent work

File: HW5_OlhaMaraduda/HW5_Task3_Olha_Maraduda.py [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# TASK_3
+# Write the script that will calculate the factorial of the entered number without using recursion.
+# Example: 0!=1, 1!=1, 2!=2, 3!=1*2*3=6.
+
```
Comment (kolyasalubov) [inline HW5_OlhaMaraduda/HW5_Task3_Olha_Maraduda.py:4]: good job
look at this version of code 
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**